app/assets/javascripts/3dbio_viewer/src/webapp/components/RootViewerContents.tsx

```diff
@@ -97,18 +108,27 @@ export const RootViewerContents: React.FC<RootViewerContentsProps> = React.memo(
                 </>
             )}
 
-            <div id="right">
-                {
+            <ResizableBox
+                width={window.innerWidth * 0.55}
+                minConstraints={[400, 0]}
+                maxConstraints={[window.innerWidth - 600, 0]}
+                axis="x"
+                resizeHandles={["w"]}
```

Comment:
All these arrays make the props non-static. As you know, they can be moved to a useMemo (when values depends on state/props) or module constant (otherwise)